### ISSUE TITLE
Follow score does not work

### ISSUE THREAD

**Issue by mmehl** [jwdj/EasyABC#42]
Enabled "Follow score", but on the score the current note is not marked.
It seems that the mark starts outside the visible window and is shifted to a wrong position.

**Comment by bomm** [comment]
I can confirm this on Ubuntu 20.04.2 LTS. I used the version https://github.com/mmehl/EasyABC/commit/3f69a41f0f99539c2aec3e98408ead376bf6c27e (fix_layout_issues branch) with Python 3.8.5, wx version 4.0.7 gtk3 (phoenix) wxWidgets 3.0.4. I can only test with python3 because with python2 I get a program crash "Fatal Python error: (pygame parachute) Segmentation Fault".
In the attached screenshot I marked the corresponding positions.
![screenshot-follow-score](https://user-images.githubusercontent.com/250905/122681237-b3e01f80-d1f3-11eb-81a4-d0040c64a3a8.png)
